metaflow/plugins/cards/card_modules/base.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
static files not included by pip install by default. Remember to add these to `setup.py` so stuff works. Currently even this fails due to missing base.html:
```
from metaflow.plugins.cards.card_client import get_cards
```

Reply:
Good point ! Great catch :D

Reply:
I have added those to the `MANIFEST`. Can pull code and try it again ?

Reply:
seems to be working now :)